calpip/Task1/Task 1-1.txt

```diff
@@ -1,21 +1,21 @@
 Dear Sir or Madam,
 
-My name is Wellington and I work voluntarily in a community organization that 
-makes a wonderful job for local children. I am writing to ask your company for
+My name is Wellington and I volunteer for a community organization that 
+provides a wonderful service for local children. I am writing to ask your company for
  a donation, which will be used to help us to organize an event next month.
 
-Although you probably have heard about us before, I would like to explain a 
-little bit about our charity. The organization which I help is called Angels 
-of the Music because we teach poor kids how to play a variety of musical 
+Although you have probably heard about us before, I would like to explain a 
+little bit about our charity. The organization I volunteer for is called Angels 
+of the Music. We teach kids from low income families how to play a variety of musical 
```

Comment:
`probably have` -> `have probably`

`which I help` sounds awkward, I think the only way to use that phrase naturally is `whom I help`

`...is called Angels of the Music because we...` should be `...is called Angels of the Music, because we...`, note the comma. But, I put a period because the sentence felt too long already. Try to use periods or commas as often as you can, it's like putting chapters to separate a long book.